picnic/build.gradle

```
@@ -19,8 +19,7 @@ compileKotlin {
 dependencies {
   implementation deps.crossword
 
-  testImplementation deps.junit
-  testImplementation deps.truth
+  testImplementation 'org.jetbrains.kotlin:kotlin-test:1.6.10'
```

The version will be picked up automatically based on the KGP version.
```suggestion
  testImplementation 'org.jetbrains.kotlin:kotlin-test'
```